src/platforms/default.rkt

```diff
@@ -6,84 +6,162 @@
+                 floor.f64
+                 fmax.f64
+                 fmin.f64
+                 fmod.f64
+                 lgamma.f64
+                 log.f64
+                 log10.f64
+                 log1p.f64
+                 log2.f64
+                 logb.f64
+                 pow.f64
+                 remainder.f64
+                 rint.f64
+                 round.f64
+                 sin.f64
+                 sinh.f64
+                 [sqrt.f64 640]
+                 tan.f64
+                 tanh.f64
+                 tgamma.f64
+                 trunc.f64)
 
-;; accelerator operations (minus fma)
-(define accelerator-platform
-  (with-terminal-cost ([binary64 64] [binary32 32])
-                      (let ([relative-costs (cost-map #:default-cost 100 [(fma) 4])])
-                        (platform-product #:optional [([real binary64])
-                                                      (cost-map-scale 64 relative-costs)]
-                                          [([real binary32]) (cost-map-scale 32 relative-costs)]
-                                          (operator-set [(real real) (erfc expm1 log1p)]
-                                                        [(real real real) (hypot)]
-                                                        [(real real real real) (fma)])))))
+(define-platform libm32-platform
+                 #:literal [binary32 32]
+                 #:default-cost 3200
+                 #:optional acos.f32
+                 acosh.f32
+                 asin.f32
+                 asinh.f32
+                 atan.f32
+                 atan2.f32
+                 atanh.f32
+                 cbrt.f32
+                 ceil.f32
+                 copysign.f32
+                 cos.f32
+                 cosh.f32
+                 erf.f32
+                 exp.f32
+                 exp2.f32
+                 [fabs.f32 64]
+                 fdim.f32
+                 floor.f32
+                 fmax.f32
+                 fmin.f32
+                 fmod.f32
+                 lgamma.f32
+                 log.f32
+                 log10.f32
+                 log1p.f32
+                 log2.f32
+                 logb.f32
+                 pow.f32
+                 remainder.f32
+                 rint.f32
+                 round.f32
+                 sin.f32
+                 sinh.f32
+                 [sqrt.f32 320]
+                 tan.f32
+                 tanh.f32
+                 tgamma.f32
+                 trunc.f32)
+
+(define-platform accelerator-platform
+                 #:literal [binary64 64]
+                 #:literal [binary32 32]
+                 #:default-cost 3200
+                 #:optional [erfc.f64 6400]
+                 [expm1.f64 6400]
+                 [log1p.f64 6400]
+                 [hypot.f64 6400]
+                 [fma.f64 256]
+                 erfc.f32
+                 expm1.f32
+                 log1p.f32
+                 hypot.f32
+                 [fma.f32 128])
```

Comment:
This should probably be folded into the other platform fragments eventually, but fine for now